style.css

```diff
@@ -51,27 +53,177 @@
   line-height: 24px;
   color: #344563;
   margin-top: 12px;
-  margin-left: 24px;
-  margin-right: 24px;
 }
 
 .details .connect {
   display: flex;
   font-weight: 500;
   font-size: 16px;
+  margin-top: 12px;
   line-height: 24px;
   color: #7f8cff;
-  margin-left: 24px;
-  margin-top: 12px;
 }
 
 .social-links {
   display: flex;
-  margin-left: 14px;
   margin-top: 12px;
 }
 
 .social-icons {
   color: #505f79;
   padding: 20px 12px;
 }
+
+/* ---------------------------works------------------------------------------ */
+
+.shadow-back {
+  background: #e5e5e5;
+  padding: 114px 24px;
+  display: grid;
+  flex-direction: column;
+  margin-top: 114px;
+  justify-content: center;
+}
+
+.works {
+  padding: 16px;
+  margin-top: 114px;
+  background: #fff;
+  width: 327px;
+  height: 552px;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
+
+h3 {
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  margin-top: 12px;
+  color: #172b4d;
+}
+
+.proj {
+  display: grid;
+  grid-template-columns: auto auto auto auto auto;
+  align-content: center;
+}
+
+.dot {
+  margin-top: 4px;
+  width: 8px;
+  height: 8px;
+  background: #c1c7d0;
+  border-radius: 50%;
+}
+
+.proj li {
+  color: #7a869a;
+  margin-right: 12px;
+  margin-top: 20px;
+  font-size: 13px;
+  line-height: 16px;
+}
+
+button {
+  margin-top: 25px;
+  padding: 5px;
+  background: #fff;
+  border: solid #396df2;
+  border-radius: 5px;
```

Comment:
- [ ] In my opinion we should stick to the design as much as much possible as recommeneded by the projec requirement, therefore we have to make sure the buttons looks the same, I suggest to reduce the border width.

  -  Your design:
![workcard](https://user-images.githubusercontent.com/97834160/205105564-28c2053f-91ad-4b32-b4ed-6d9a40119c67.png)


  - Figma design:
  
![figmaWorkCARD](https://user-images.githubusercontent.com/97834160/205105673-7133d7a6-4c03-4a07-a925-c72d3ae3592b.png)